### PR TITLE
[release-v1.15] Run `make generate-release`

### PR DESCRIPTION
**Pull request by serverless-qe**
Run `make generate-release` using openshift-knative/hack.

### CODE REVIEW

File: openshift/ci-operator/build-image/Dockerfile [hunk 1]
```diff
@@ -22,6 +23,3 @@ RUN GOFLAGS='' go install github.com/mikefarah/yq/v3@latest
 # go install creates $GOPATH/.cache with root permissions, we delete it here
 # to avoid permission issues with the runtime users
 RUN rm -rf $GOPATH/.cache
-
-# Allow runtime users to add entries to /etc/passwd
```
Comment (ReToCode) [inline openshift/ci-operator/build-image/Dockerfile:26]: hm, no idea why that was here in the first place. Ideas @skonto @mgencur?

Comment (pierDipi) [reply openshift/ci-operator/build-image/Dockerfile:26]: it's a legacy setting used only in SO Dockerfiles and we don't need it anymore

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

**Comment by skonto** [comment]
```
Error from server (InternalError): error when creating "./hack/lib/certmanager_resources/serving-selfsigned-issuer.yaml": Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "https://cert-manager-webhook.cert-manager.svc:443/validate?timeout=30s": tls: failed to verify certificate: x509: certificate signed by unknown authority
13:02:27.562 ERROR:   🚨 Error (code: 1) occurred at ./hack/lib/certmanager.bash:7, with command: return $?
13:02:27.566 INFO:    Dumping state...
```

**Comment by ReToCode** [comment]
We can override the TLS tests, as they'll be covered with https://github.com/openshift-knative/serving/pull/801 (I'm working on that).

File: openshift/ci-operator/build-image/Dockerfile (hunk 1 above)
Comment (skonto) [reply openshift/ci-operator/build-image/Dockerfile:26]: It shoul dbe moved to the CI config or something, see [here](https://redhat-internal.slack.com/archives/CD87JDUB0/p1726045452787339).

**Pull request merged**